docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,190 @@
+  2. Creating a PMK Cluster.
+  3. Connect the cluster to Microsoft Azure Arc.
+
+**Onboarding the node to Platform9 Management Plane**
+
+- Login to your Management Plane.
+
+  ![Screenshot of PMK Management Plane Login Page](./04.png)
+
+  > **Note: If you do not have a registered Management Plane with Platform9, you can create one easily using [PMK Free Tier deployment](https://platform9.com/managed-kubernetes/)**
+
+- Onboard the Ubuntu host to your Management Plane.
+
+  For a BareOS cluster, you will need to have the nodes registered with the PMK Management Plane on which the cluster is to be deployed. For this, first add the node.
+
+  ![Screenshot of Onboarding a node](./05.png)
+
+  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to setup the nodes and get connected with Management Plane.
+  Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
+
+  ![Screenshot of PF9 CLI](./06.png)
+
+  > **Note: Preparing the node and connecting it to Management Plane might take approximately 4-5 minutes to complete.**
+
+
+**Creating a PMK Cluster**
+
+- The onboarded node should now be seen on our Management Plane. Now let's create a [PMK cluster](https://platform9.com/learn/learn/get-started-bare-metal) using that onboarded node.
+
+  The cluster creation is done from the PMK Management Plane UI. The steps for cluster creation will follow as below,
+
+  - Click to add cluster to the Management Plane.
+
+  ![Screenshot of Adding Cluster](./07.png)
+
+  - Create a cluster from the nodes onboarded to the Management Plane.
+
+  ![Screenshot of Creating One Click Cluster](./08.png)
+
+  - Enter the cluster name and k8s version. Submit the page and it will trigger the cluster creation on the node.
+
+  ![Screenshot of Cluster name entry](./09.png)
+
+  - The cluster should get created in a few minutes and the status should be reported as **Healthy**.
+
+  ![Screenshot of Cluster Created](./10.png)
+
+**Connect the cluster to Microsoft Azure Arc**
+
+- Clone the [azure_arc](https://github.com/deepuks/azure_arc) Jumpstart repository and change to the pf9 script directory.
+
+  ```shell
+  git clone https://github.com/microsoft/azure_arc.git
+  cd azure_arc/azure_arc_k8s_jumpstart/pf9
+  ```
+
+- Set the variables according to your Azure service principal name and Azure environment in the *pf9_az.env* file.
+
+  ![Screenshot of Exporting values](./11.png)
+
+  > **Note: The values can referenced from the service principal and resource groups outputs.**
+
+- Set the KUBECONFIG variable for connecting to the K8s cluster. The kubeconfig file for the cluster is available for download from the Management Plane.
+
+  ![Screenshot to Download Kubeconfig](./13.png)
+
+  - An example output is shown below for setting the KUBECONFIG variable;
+
+  ![Screenshot to Export KUBECONFIG](./14.png)
```

Comment:
What this screenshot is suppose to show in the context of the guide?

Reply:
This has been removed and changed to include in shell.